src/views/Correction.vue

```diff
@@ -192,17 +206,16 @@ export default class Correction extends Mixins(DateMixin, FilingTemplateMixin, L
         if (correctionFiling.header.status !== FilingStatus.DRAFT) {
           throw new Error('Invalid Correction status')
         }
-        // fetch original IA to correct
-        const correctedFilingID = correctionFiling.correction?.correctedFilingId
-        const correctedFiling = await this.fetchFilingById(correctedFilingID)
-        this.setCorrectedFilingId(correctedFilingID)
 
-        // parse IA filing into store
-        // this is the initial state of the correction filing
-        this.setOriginalIA(correctedFiling)
+        // get and store ID of filing that is being corrected (ie, original IA)
```

Comment:
@cameron-freshworks I did not change load order here (though we talked about it). The new idea of saving action flags (added/edited/removed) in the draft means load order to check for changes is not needed.